apps/site/components/withNavBar.tsx

```diff
@@ -16,13 +17,23 @@ const WithNavBar: FC = () => {
   const { replace } = useRouter();
   const pathname = usePathname();
 
+  const skipToContent = classNames(
+    'bg-[#000] text-center font-semibold inline-flex items-center justify-center gap-2 py-2.5 motion-safe:transition-colors rounded border border-green-600 bg-green-600 text-white shadow-sm',
+    'absolute left-0 top-0 m-3 -translate-y-16 bg-blue-500 p-3 text-white transition-transform focus:translate-y-0 focus:outline-none'
+  );
+
   const locale = useLocale();
+  const t = useTranslations();
 
   const toggleCurrentTheme = () =>
     setTheme(resolvedTheme === 'dark' ? 'light' : 'dark');
 
   return (
     <div>
+      <a className={skipToContent} href="#main">
```

Comment:
```suggestion
      <Button className={skipToContent} href="#main">
```

use the component button `@/components/common/Button` it's already exist and use our design system

Reply:
Yes, the Button component already exists, but there's a problem with this - https://github.com/nodejs/nodejs.org/pull/7390#discussion_r1912119436

Reply:
our `<button>` is a `<a>` under the hood

Reply:
so just add `classname` props to add more tailwind css classes need to positioned the button

Reply:
Pushed a commit for the same. I've changed something in the index.module.css file (Button.tsx), will it be valid?